stdlib/public/Platform/README.md

```diff
@@ -1,11 +1,8 @@
-This directory holds the Swift Standard Library's Darwin and Glibc Module,
+This directory holds the Swift Standard Library's Glibc Module,
 comprised of
 
 - The *overlay* library, which amends some APIs imported from Clang module.
-- Glibc only: The clang [module map] which specifies which headers need to
-  be imported from Glibc for bare minimum functionality.
-
-Note: On Darwin platforms, we assume clang [module map] is already installed
-in the system SDK.
```

Comment:
Optional: I suggest to leave this mention (or similar) in this file, so readers know that we are not building any overlay for Darwin platforms on purpose.